client/src/App.js

```diff
@@ -59,6 +60,11 @@ function App() {
         <PrivateRoute path="/checkout">
           <Checkout />
         </PrivateRoute>
+        <Route path="/404">
+          <PageNotFound />
+        </Route>
+        <Route exact path="/" component={Home} />
+        {/* <Redirect from="*" to="/404" /> */}
```

Comment:
```suggestion
        <Route component={PageNotFound} />
```
This seems to work for me! The wildcard * in the from/to will match all routes, but I think this will just match everything else.

Found in [this article](https://ultimatecourses.com/blog/react-router-not-found-component) although we're not using switch here